lerna.json

```diff
@@ -6,7 +6,7 @@
   "command": {
     "publish": {
       "ignoreChanges": ["*.md", "docs"],
-      "message": "chore(release): publish %s"
+      "message": "chore(release): publish"
```

Comment:
Lerna won't send the version if the monorepo is set to `"version": "independent"` which we are.